backup/backup.go

```
@@ -189,12 +191,14 @@ func DoBackup() {
 		connectionPool.MustCommit(connNum)
 	}
 	metadataFile.Close()
-	if MustGetFlagString(utils.PLUGIN_CONFIG) != "" {
+	if pluginConfigFlag != "" {
 		pluginConfig.MustBackupFile(metadataFilename)
 		pluginConfig.MustBackupFile(globalFPInfo.GetTOCFilePath())
 		if MustGetFlagBool(utils.WITH_STATS) {
 			pluginConfig.MustBackupFile(globalFPInfo.GetStatisticsFilePath())
 		}
+		utils.CopyFile(pluginConfigFlag, globalFPInfo.GetPluginConfigPath())
```

It is possible for `CopyFile(...)` to return an error. If we are intentionally ignoring this error can we change this line to `_ = utils.CopyFile(...)` so we the fact that something is being returned.